tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -155,6 +168,19 @@ int GetBuiltinOperatorVersion(const OpSignature& op_sig) {
       // | Quantized Int8  |                  4 |                        4 |
       // +-----------------+--------------------+--------------------------+
 
+      auto fully_connected_params =
+          reinterpret_cast<TfLiteFullyConnectedParams*>(op_sig.builtin_data);
+      TFLITE_DCHECK(fully_connected_params != nullptr);
+
+      if (op_sig.inputs.at(0).type == kTfLiteInt16 &&
+          op_sig.inputs.at(1).type == kTfLiteInt8 &&
+          op_sig.outputs.at(0).type == kTfLiteInt16) {
+        // `quantized_bias_type` is supported at version 10.
+        if (fully_connected_params->quantized_bias_type) {
+          return 10;
```

Comment:
Where is op version 9?

Reply:
op version 9 seems to be here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/versioning/op_version.cc#L204 
Please note, I had to update the op version to 11 as it seems op version 10 has been added. 
Please let me know if I should not do this. Thanks.

Reply:
SGTM